src/types.d.ts

```diff
@@ -49,3 +54,10 @@ export type ScrollableRef = {
   type: ScrollableType;
 };
 //#endregion
+
+export interface NullableAccessibilityProps extends AccessibilityProps {
```

Comment:
There are some use cases where the user would like to set other accessibility props as well (e.g. accessibilityState, accessibilityValue, onAccessibilityTap, accessibilityActions)

Reply:
okay , then why you redefined `accessible`, `accessibilityLabel`... , they are already in `AccessibilityProps` ?

Reply:
That is to make them overwritable. To avoid breaking changes it will work the same for old users but those who want the screen reader to not read out these properties at all can pass a null

Reply:
i think `undefined` should work fine ? right

Reply:
I made it to apply default values if nothing is defined (this avoids breaking changes)

Right now if property is undefined = apply default value
If property is null = apply undefined value

Reply:
If i pass undefined as a value it eill still apply default english values

Reply:
ah, okay